### ISSUE TITLE
Crash on sending a message with emojis

### ISSUE THREAD

**Issue by ggoraa** [mock-foundation/moc#131]
Possibly the cause is an old version of TDLib with an old API layer, but the thing is that this should've been handled 🧐

**Comment by ggoraa** [comment]
Nope, the cause is something in TDLibKit (Swift wrapper around TDLib), investigating here: https://github.com/Swiftgram/TDLibKit/issues/17

**Comment by voltangle** [comment]
This thing fixed itself lmao
probably because i dropped macOS 12
closing
